src/routes/landing/components/FilterableCitySelector.js

```diff
@@ -17,34 +19,70 @@ type PropsType = {
 }
 
 type StateType = {
-  filterText: string
+  filterText: string,
+  currentLongitude: number | null,
+  currentLatitude: number | null,
+  renderLocationList: boolean
```

Comment:
This bool is redundant: We should always render the location list, if `currentLongitude` and `currentLatitude` are available (aka non null).
You can remove it.